gradle.properties

```diff
@@ -0,0 +1,5 @@
+org.gradle.daemon=true
+org.gradle.caching=true
```

Comment:
This doesn't seem like it would help us at all. If it helps you, add it to your user-specific ~/.gradle/gradle.properties.